csrc/mmdeploy/core/CMakeLists.txt

```diff
@@ -65,7 +65,7 @@ if (MMDEPLOY_BUILD_SDK_CXX_API)
 endif ()
 
 target_link_libraries(${PROJECT_NAME} PUBLIC ${SPDLOG_LIB})
-if (NOT (MSVC OR ANDROID OR APPLE))
+if (NOT (MSVC OR ANDROID OR APPLE OR (CMAKE_C_COMPILER MATCHES "aarch64")))
```

Comment:
I think the method from 'pybind11/tests/CMakeLists.txt' to set `STD_FS_LIB` is better than ours. @lzhangzz what's your opinion?

Reply:
#1190